atomistics/shared/output.py

```diff
@@ -77,26 +77,6 @@ class OutputEnergyVolumeCurve(
     volume: callable
 
 
-@dataclasses.dataclass
-class OutputElastic(Output):
-    elastic_matrix: callable
-    elastic_matrix_inverse: callable
-    bulkmodul_voigt: callable
-    bulkmodul_reuss: callable
-    bulkmodul_hill: callable
-    shearmodul_voigt: callable
-    shearmodul_reuss: callable
-    shearmodul_hill: callable
-    youngsmodul_voigt: callable
-    youngsmodul_reuss: callable
-    youngsmodul_hill: callable
-    poissonsratio_voigt: callable
-    poissonsratio_reuss: callable
-    poissonsratio_hill: callable
-    AVR: callable
-    elastic_matrix_eigval: callable
```

Comment:
While currently only the ElasticMatrix implements the calculation of the elastic moduli, @samwaseda developed the ElasticTensor class which can also calculate the elastic moduli. https://github.com/pyiron/pyiron_atomistics/blob/main/pyiron_atomistics/atomistics/master/elastic.py So I would prefer to have a reference interface for the elastic moduli defined in `shared/output` and a specific implementation in the `workflows/elastic/workflow.py` module.

Reply:
I would suggest to make it specific now anyhow, then introduce a shared parent if and when failing to do so would result in code duplication or a lack of clarity about the relationship between the two approaches.